PowerUp/app/src/main/java/powerup/systers/com/MainActivity.java

```diff
@@ -12,6 +14,15 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
+		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
+		boolean previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
+		if(!previouslyStarted) {
+			SharedPreferences.Editor edit = prefs.edit();
+			edit.putBoolean(getString(R.string.pref_previously_started), Boolean.TRUE);
+			edit.commit();
```

Comment:
You should probably not store the previously started true value until the user completes the changing room and proceeds to the map, so in case the action stops in the middle, you don't leave an inconsistent state